lib/util/i18n.js

```diff
@@ -208,6 +211,62 @@ export function getFormattedMode(mode, intl) {
   }
 }
 
+/**
+ * Returns a FormattedMessage string for the configured access mode via the react-intl imperative API
+ * such that i18n IDs are hardcoded and can be kept track of by format.js CLI tools
+ */
+export function getFormattedConfiguredAccessMode(mode, intl) {
+  switch (mode?.toLowerCase()) {
+    case 'bicycle':
+      return intl.formatMessage({ id: 'config.accessModes.bicycle' })
+    case 'bicycle_rent':
+      // TODO: support company names so this reads e.g. "Transit + BCycle"
+      return intl.formatMessage({ id: 'config.accessModes.bicycle_rent' })
+    case 'car_hail':
+      return intl.formatMessage({ id: 'config.accessModes.car_hail' })
+    case 'car_park':
+      return intl.formatMessage({ id: 'config.accessModes.car_park' })
+    case 'micromobility':
+      return intl.formatMessage({ id: 'config.accessModes.micromobility' })
+    case 'micromobility_rent':
+    case 'scooter_rent':
```

Comment:
```suggestion
    case 'scooter_rent':
    case 'scooter':
```

Still need to double check with Leonard but I believe `scooter_rent` does not exist, although it seems to have an effect when used in a query. However, `scooter` is the main scooter mode

Reply:
`SCOOTER_RENT` is used in one of the configs. Should it be deleted from there?

Reply:
It's been removed (as part of other scooter fixes) in https://github.com/ibi-group/configurations/pull/257